src/twisted/web/newsfragments/9421.bugfix

```diff
@@ -0,0 +1 @@
+twisted.web.client.Request and twisted.web.client.HTTPClient were both vulnerable to header injection attacks.  They now replace linear whitespace ('\r', '\n', and '\r\n') with a single space.
```

Comment:
Maybe this can make a reference to https://tools.ietf.org/html/rfc2616#section-4.2 in the case in which someone will ask why are we doing this and not raising an error.